tests/framework/db/SchemaTest.php

```diff
@@ -230,7 +230,7 @@ public function getExpectedColumns()
                 'size' => null,
                 'precision' => null,
                 'scale' => null,
-                'defaultValue' => 1.23,
+                'defaultValue' => '1.23',
```

Comment:
would be good if we could avoid this change. i.e. there is no need to fix this in the db->php direction.